alacritty/src/display/mod.rs

```diff
@@ -548,11 +562,15 @@ impl Display {
     fn clear_glyph_cache(&mut self) {
```

Comment:
```suggestion
    fn reset_glyph_cache(&mut self) {
```

Just for consistency, the docs probably should also be updated.